webclient/core/views.py

```diff
@@ -134,81 +147,94 @@ def upload_file_samostatny_nalez(request, ident_cely):
 
 @require_http_methods(["POST"])
 def post_upload(request):
-    logger.debug("Uploading file to object: " + request.POST["objectID"])
-    projects = Projekt.objects.filter(ident_cely=request.POST["objectID"])
-    documents = Dokument.objects.filter(ident_cely=request.POST["objectID"])
-    finds = SamostatnyNalez.objects.filter(ident_cely=request.POST["objectID"])
-    typ_souboru = ""
-    if projects.exists():
-        objekt = projects[0]
-        typ_souboru = OTHER_PROJECT_FILES
-        new_name = get_projekt_soubor_name(request.FILES.get("file").name)
-    elif documents.exists():
-        objekt = documents[0]
-        typ_souboru = OTHER_DOCUMENT_FILES
-        new_name = get_dokument_soubor_name(objekt, request.FILES.get("file").name)
-    elif finds.exists():
-        objekt = finds[0]
-        typ_souboru = PHOTO_DOCUMENTATION
-        new_name = get_finds_soubor_name(objekt, request.FILES.get("file").name)
+    update = "fileID" in request.POST
+    if not update:
```

Comment:
@jnihnat Honzo, mrkni se prosím na tuto úpravu. Řešil jsem tu konflikt a nevím, jestli jsem při tom něco nerozbil. Díky.

Reply:
@pesikj vypada ze to tam ostalo cele, az mergnes prosim napis, stahnu si dev a otestuju u sebe.